src/RemoteTech/API/API.cs

```diff
@@ -10,11 +10,12 @@ public static class API
     {
         public static bool HasLocalControl(Guid id)
         {
-            var satellite = RTCore.Instance.Satellites[id];
-            if (satellite == null) return false;
-            RTLog.Verbose("Flight: {0} HasLocalControl: {1}", RTLogLevel.API, id, satellite.HasLocalControl);
+            var vessel = FlightGlobals.Vessels.FirstOrDefault(v => v.id == id);
+            if (vessel == null) return false;
+
+            RTLog.Verbose("Flight: {0} HasLocalControl: {1}", RTLogLevel.API, id, vessel.HasLocalControl());
 
-            return satellite.HasLocalControl;
+            return vessel.HasLocalControl();
```

Comment:
bypassing the core was the easiest way to be sure we could always get the vessel.